pico-setup fails on fresh RPi install as of Nov 10.

Hello Everyone,

This script has started failing consistency on new installations. I believe this is tied to [this merge](https://github.com/raspberrypi/picoprobe/pull/31) in the picoprobe repo.

Here is the section of the terminal output where it fails. (I am not including everything above it. The first line below is the final line of the hello_usb build which works. 

```
[100%] Built target hello_usb
Cloning into 'picoprobe'...
remote: Enumerating objects: 155, done.
remote: Counting objects: 100% (155/155), done.
remote: Compressing objects: 100% (67/67), done.
remote: Total 155 (delta 98), reused 131 (delta 85), pack-reused 0
Receiving objects: 100% (155/155), 44.72 KiB | 1.79 MiB/s, done.
Resolving deltas: 100% (98/98), done.
Using PICO_SDK_PATH from environment ('/home/pi/pico/pico-sdk')
PICO_SDK_PATH is /home/pi/pico/pico-sdk
Defaulting PICO_PLATFORM to rp2040 since not specified.
Defaulting PICO platform compiler to pico_arm_gcc since not specified.
-- Defaulting build type to 'Release' since not specified.
PICO compiler is pico_arm_gcc
CMake Error at FreeRTOS_Kernel_import.cmake:57 (message):
  Directory '/home/pi/pico/picoprobe/freertos' does not contain an RP2040
  port here: portable/ThirdParty/GCC/RP2040
Call Stack (most recent call first):
  CMakeLists.txt:6 (include)


-- Configuring incomplete, errors occurred!
$
```

My workaround has been to remove the picoprobe install by changing [line 94](https://github.com/raspberrypi/pico-setup/blob/master/pico_setup.sh#L94) from `for REPO in picoprobe picotool` to `for REPO in picotool`.

Obviously this is ok acceptable for people who don't need picoprobe. Since we don't use picoprobe, I didn't look into what may have changed in that repo, but I wanted to call attention here that I believe this script is broken at the moment.

Thanks!
Brian

--- a/pico_setup.sh
+++ b/pico_setup.sh
@@ -99,6 +99,7 @@ do
 
     # Build both
     cd $DEST
+    git submodule update --init
     mkdir build
     cd build
     cmake ../

